File: Sai Ganesh/day1/palindrome.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        rev=0
        y=x
        if x<0 :
            return "false"
        while(x!=0):
            n=x%10
            rev=rev*10+n
            x=x//10
        if rev==y:
            return "true"
        return "false"

File: Nithya/Week-1/Palindrome.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long int r,sum=0,original;
        original=x;
        while(x>0)
        {
            r=x%10;
            sum=(sum*10)+r;
            x=x/10;
        }
        if(original==sum)
        return true;
        else
            return false;
        
        
    }
};